<launch>

  <!-- recognition -->
  <group ns="stereo_preprocessing">
    <include file="$(find drc_task_common)/launch/fc/stereo_preprocess.launch">
      <arg name="INPUT_POINT_CLOUD" value="/multisense/resize_1_4/points" />
    </include>
  </group>
  <node pkg="image_view2" type="image_view2" name="image_view2" 
        >
    <remap from="image" to="/multisense/left/image_rect_color" />
    <remap from="/multisense/left/image_rect_color/screenrectangle" to="/drill_recognition_for_put/input_plane_rect" />
  </node>
  <include file="$(find drc_task_common)/launch/fc/drill_recognition_for_put.launch">
    <arg name="INPUT_HINT_POINT_CLOUD" value="/stereo_preprocessing/normal_estimation/output_with_xyz" />
    <arg name="INPUT_POINT_CLOUD" value="/stereo_preprocessing/normal_estimation/output_with_xyz" />
  </include>
  
</launch>
